diff --git a/res/freedesktop/com.solvespace.SolveSpace.metainfo.xml.in b/res/freedesktop/com.solvespace.SolveSpace.metainfo.xml.in
index 9e49ecc8..5a16122f 100644
--- a/res/freedesktop/com.solvespace.SolveSpace.metainfo.xml.in
+++ b/res/freedesktop/com.solvespace.SolveSpace.metainfo.xml.in
@@ -67,6 +67,14 @@
   <content_rating type="oars-1.0" />
 
   <releases>
+    <release version="3.2" date="2026-03-27" type="stable">
+      <description>
+        <p>Major new stable release. Includes parallel and perpendicular constraints for faces,
+        expands many existing constraints to operate on more than two entities at once. Basic
+        kerning in text elements. STEP files export as solid models. A variety of UI improvements.</p>
+      </description>
+      <url>https://github.com/solvespace/solvespace/releases/tag/v3.2</url>
+    </release>
     <release version="3.1" date="2022-06-01" type="stable">
       <description>
         <p>Major new stable release. Includes new arc and line length ratio and difference
@@ -78,7 +86,7 @@
         files into a model. Includes a variety of UI improvements. Speeds up complex sketches
         by up to 8x and doubles the maximum unknowns.</p>
       </description>
-      <url>https://github.com/solvespace/solvespace/releases/tag/v3.0</url>
+      <url>https://github.com/solvespace/solvespace/releases/tag/v3.1</url>
     </release>
     <release version="3.0" date="2021-04-18" type="stable">
       <description>
